api-report/driver-utils.api.md

```diff
@@ -206,6 +208,9 @@ export interface IProgress {
 // @internal
 export function isCombinedAppAndProtocolSummary(summary: ISummaryTree | undefined): summary is CombinedAppAndProtocolSummary;
 
+// @public (undocumented)
+export function isDeltaStreamConnectionForbiddenError(error: any): error is DeltaStreamConnectionForbiddenError;
```

Comment:
i don't think we need this outside of conatiner, so it would be better to just put it there and not export it for now.